src/twisted/trial/__main__.py

```diff
@@ -4,6 +4,15 @@
 if __name__ == "__main__":
     import sys
 
-    from pkg_resources import load_entry_point
+    if sys.version_info >= (3, 8):
```

Comment:
Sigh. Okay. Codecov is too smart for our trickery.

Which actually raises the question: why are we doing this faffing with packaging metadata, anyway?  We used to do this because we wanted to be as similar as possible to the stubs that `pip install` generated, but looking at a modern virtualenv, I see the stubs look like this:

```
#!/Users/glyph/.virtualenvs/Twisted/bin/python
# -*- coding: utf-8 -*-
import re
import sys
from twisted.scripts.trial import run
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(run())
```

So that implies that this should instead simply be:

```
    from twisted.scripts.trial import run
    sys.exit(run())
```

Is there any reason not to do that?

(Suggestion in multiple parts below)

Reply:
very true